Gemfile

```diff
@@ -11,7 +10,7 @@ gem 'rails', '3.2.22.2'
 
 # Database
 # gem 'sqlite3-ruby', :require => 'sqlite3'
-gem 'mysql2', '0.3.10'
+gem 'mysql2', '0.3.20'
```

Comment:
didn't we already change mysql2 to something higher? I might be totally misremembering...